components/centraldashboard/public/components/main-page.pug

```diff
@@ -45,7 +46,8 @@ app-drawer-layout.flex(narrow='{{narrowMode}}',
             neon-animatable(page='activity')
                 activity-view(namespace='[[queryParams.ns]]')
             neon-animatable#iframe-page(page='iframe')
-                iframe#PageFrame.flex(src='[[iframeUrl]]')
+                iframe#PageFrame.flex(src='[[iframeUrl]]',
+                    on-load='_sendNamespaceMessage')
```

Comment:
Don't need this in adherence to new design

Reply:
Ack'ed